src/main/java/org/broadinstitute/hellbender/tools/copynumber/formats/collections/SimpleCountCollection.java

```diff
@@ -27,16 +27,16 @@
     /**
      * CONTIG, START, END, COUNT
      */
-    enum SimpleCountTableColumn {
+    public enum SimpleCountTableColumn {
```

Comment:
Actually, I think it'll be easier just to duplicate these privately in the tool for the time being.  Otherwise, we have to remember to go back and fix permissions.

Reply:
Why the column composition of the count files should be a private matter? Perhaps I could move my class the enclosing package to be able to access. Duplicate a enum and make sure they keep consistent is not a good idea.

Reply:
"Duplicate...for the time being"!  Easier to fix the idiosyncratic code here later, rather than remember what we changed in the CNV package.

The file is a 1:1 representation of the Java class and any code that needs to talk to it should be encapsulated in this package, if not the class itself.